docs/quality-assurance/Web-Accessibility.md

```diff
@@ -0,0 +1,26 @@
+# Web Accessibility Testing
```

Comment:
I feel this page dives straight in without any background on what is meant by "accessibility" and what the purpose of accessibility testing is.

I feel the document should have the following sections:
 - What is accessibility
 - What is accessibility testing
 - Why do we need it
 - How to approach automated testing
 - What tools are available

Then anyone with an interest will have all the information they need to get going. I don't feel the content as it stands meets this requirement.